realm/realm-library/src/androidTestObjectServer/java/io/realm/SyncConfigurationTests.java

```diff
@@ -61,12 +62,17 @@
     @Rule
     public final ExpectedException thrown = ExpectedException.none();
 
+    @Before
+    public void setUp() {
+        Realm.init(InstrumentationRegistry.getTargetContext());
+    }
+
     @After
     public void tearDown() {
-        for (SyncUser syncUser : SyncUser.all().values()) {
-            syncUser.logOut();
+        UserStore userStore = SyncManager.getUserStore();
+        for (SyncUser syncUser : userStore.allUsers()) {
+            userStore.remove(syncUser.getIdentity(), syncUser.getAuthenticationUrl().toString());
         }
-        SyncManager.reset();
```

Comment:
why removing this? are you trying to avoid the `no_active_sessions` assertion in OS?

Reply:
Yes, The `SyncConfigurationTests` don't actually log users in but fake it by creating a fake server response without touching the server. This means Object Store isn't initialized correctly resulting in a lot of weird errors.